index.js

```diff
@@ -1,663 +1,246 @@
-            }
-        },
-
-        {
-            '38':   {
-                en: ['Up','Up'],
-                ru: ['Up','Up'],
-            }
-        },
-
-        {
-            '16':   {
-                en: ['Shift','Shift'],
-                ru: ['Shift','Shift'],
-            }
-        },
-    ],
-
-    [
-        {
-            '17':   {
-                en: ['Ctrl','Ctrl'],
-                ru: ['Ctrl','Ctrl'],
-            }
-        },
-        {
-            '91':   {
-                en: ['Win','Win'],
-                ru: ['Win','Win'],
-                
-            }
-        },
-        {
-            '18':   {
-                en: ['Alt','Alt'],
-                ru: ['Alt','Alt'],
-            }
-        },
-        {
-            '32':   {
-                en: [' ',' '],
-                ru: [' ',' '],
-            }
-        },
-        {
-            '18':   {
-                en: ['Alt','Alt'],
-                ru: ['Alt','Alt'],
-            }
-        },
-        {
-            '17':   {
-                en: ['Ctrl','Ctrl'],
-                ru: ['Ctrl','Ctrl'],
-            }
-        },
-        {
-            '37':   {
-                en: ['Left','Left'],
-                ru: ['Left','Left'],
-            }
-        },
-        {
-            '40':   {
-                en: ['Down','Down'],
-                ru: ['Down','Down'],
-            }
-        },
-        {
-            '39':   {
-                en: ['Right','Right'],
-                ru: ['Right','Right'],
-            }
-        },
-    ],
-]
-// import {keyboard} from './src/data.js';
+// eslint-disable-next-line import/extensions
+import keyboard from './src/data.js';
 
 let currentLang = localStorage.getItem('lang') || 'en';
 let cups = false;
 const input = document.createElement('textarea');
-const root = document.createElement('div'); 
+const root = document.createElement('div');
 root.id = 'root';
 input.id = 'keyboard-input';
 document.body.append(input);
 document.body.append(root);
 input.disabled = true;
 
 // Creature DOM
-keyboard.forEach(function(row) {
-    const rowWraper = document.createElement('div');
-    row.forEach(function(buttonObj) {
-        const key = Object.keys(buttonObj)[0];
-        const button = document.createElement('button');
-        button.id = key;
-        button.setAttribute('uniqueAttribute', buttonObj[key][currentLang][0]);
-        button.classList.add('button');
-        button.innerHTML = buttonObj[key][currentLang][0];
-        rowWraper.appendChild(button);
-    });
-    root.appendChild(rowWraper);
+keyboard.forEach((row) => {
+  const rowWraper = document.createElement('div');
+  row.forEach((buttonObj) => {
+    const key = Object.keys(buttonObj)[0];
+    const button = document.createElement('button');
+    button.id = key;
+    button.setAttribute('uniqueAttribute', buttonObj[key][currentLang][0]);
+    button.classList.add('button');
+    const [lower] = buttonObj[key][currentLang];
```

Comment:
buttonObj[key][currentLang][0] is the same as const [lower] = buttonObj[key][currentLang];, You can perform this operation once.